hadoop-ozone/dist/src/shell/conf/log4j.properties

```diff
@@ -15,7 +15,7 @@
 # limitations under the License.
 
 # Define some default values that can be overridden by system properties
-hadoop.root.logger=INFO,console
+hadoop.root.logger=INFO,console,ASYNCRFA
```

Comment:
One question here: should we really add this new appender by default?  Regular RFA is only defined, but not set.

Reply:
Yes, we could remove `ASYNCRFA` from default and override it via system properties when required. Made the required changes, thank you @adoroszlai!